meta-balena-common/recipes-connectivity/networkmanager/networkmanager_1.20.2.bb

```diff
@@ -19,7 +19,7 @@ DEPENDS = " \
     curl \
 "
 
-inherit gnomebase gettext systemd bluetooth bash-completion vala gobject-introspection gtk-doc
+inherit gnomebase gettext systemd bash-completion vala gobject-introspection gtk-doc ${@bb.utils.contains('DISTRO_CODENAME', 'dunfell', '', 'bluetooth', d)}
```

Comment:
As before, please remove the bluetooth inheritance from meta-balena-common and add it to compatibility layers as needed.

Reply:
I hardcoded the fix in common and added compatibility workarounds to warrior and thud. Should I add the compatibility to older layers as well?

Reply:
Yep, see above.